tests/test_bdist_wheel.py

```diff
@@ -14,7 +14,8 @@
     'dummy_dist-1.0.dist-info/RECORD'
 }
 DEFAULT_LICENSE_FILES = {
-    'LICENSE', 'LICENSE.txt', 'LICENCE', 'LICENSE.txt', 'COPYING', 'NOTICE'
+    ('LICENSE', 'LICENSE.txt', 'LICENCE', 'LICENSE.txt', 'COPYING',
+     'COPYTING.md', 'NOTICE', 'NOTICE.rst', 'AUTHORS', 'AUTHORS.txt')
 }
```

Comment:
You just changed the declaration from a set of strings to a set containing only one tuple.

Reply:
Sorry, stupid mistake; thanks for the catch. The other place I changed it *was* an implicit tuple, and I didn't parse the braces properly at first glance. Fixed.